Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -994,6 +1028,8 @@ - (BOOL)prefersStatusBarHidden
 #endif
 
     m_page->send(Messages::VideoFullscreenManager::SetVideoLayerFrameFenced(contextId, frame, fenceSendRight));
+
+    [interface->captionsLayer() setFrame:CGRectMake(0, 0, frame.width(), frame.height())];
```

Comment:
...because this should be the responsibility of the LayerHostView to perform when is own size changes.